lms/static/js/Markdown.Editor.js

```diff
@@ -1035,6 +1040,28 @@
             }
         };
 
+        // Add error msg in hyperlink input box if there isn't any error msg already
+        var addFormError = function (isImage){
+            if (!dialog.contains(formError)) {
+                formError = doc.createElement("span");
+                style = formError.style;
+                style.width = "85%";
+                style.display = "inline-block";
+                style.fontSize = "85%";
+                style.marginLeft = "30px";
+                var errorMsg;
+                if (isImage) {
+                    errorMsg = doc.createTextNode("You either have to give description or select checkbox");

```

Comment:
Let's make the error message="Please describe this image or agree that it has no contextual value by checking the checkbox"